operators/pkg/instctrl/cloudinit.go

```diff
@@ -86,6 +87,33 @@ func (r *InstanceReconciler) EnforceCloudInitSecret(ctx context.Context) error {
 	return nil
 }
 
+// Get the specs of the volume
```

Comment:
This comments looks difficult to understand to me, unless one already knows what you are doing.

Reply:
Added clearer comment